pkg/kubelet/kubeletconfig/util/codec/codec.go

```diff
@@ -50,8 +50,8 @@ func NewJSONEncoder(groupName string) (runtime.Encoder, error) {
 }
 
 // DecodeKubeletConfiguration decodes an encoded (v1alpha1) KubeletConfiguration object to the internal type
-func DecodeKubeletConfiguration(data []byte) (*componentconfig.KubeletConfiguration, error) {
-	// TODO(mtaufen): when KubeletConfiguration moves out of componentconfig, will the UniversalDecoder still work?
+func DecodeKubeletConfiguration(data []byte) (*kubeletconfig.KubeletConfiguration, error) {
```

Comment:
Can you link to the kube-proxy code you're referencing?

Did you mean s/cm/cmd? Not sure I want this in cmd/, as the kubelet config libs in pkg/ depend on it...